code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -313,7 +313,7 @@ GLOBAL_LIST_INIT(titanium_recipes, list ( \
 	walltype = /turf/closed/wall/mineral/plastitanium
 
 GLOBAL_LIST_INIT(plastitanium_recipes, list ( \
-	new/datum/stack_recipe("plastitanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20, category = CAT_TILES), \
+	new/datum/stack_recipe("plastitanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20, check_density = FALSE, category = CAT_TILES), \
 	))
```

Comment:
fwiw these backslashes are completely redundant since DM allows multilining lists even if they are in a macro wrapper (metal hydrogen on 466 proves this) you don't have to change anything but i wanted to mention it here because of how common this pattern is